doctor/docs/base.py

```diff
@@ -11,9 +11,9 @@
 import six
 try:
     from docutils import nodes
+    from docutils.parsers.rst import Directive
     from docutils.statemachine import ViewList
     from sphinx.errors import SphinxError
-    from sphinx.util.compat import Directive
     from sphinx.util.nodes import nested_parse_with_titles
```

Comment:
Was getting a warning about this import being removed in 1.7.x so changed it to use docutils.